chain/client/src/test_utils.rs

```diff
@@ -1270,6 +1283,36 @@ impl TestEnv {
         self.process_block(id, block.unwrap(), Provenance::PRODUCED);
     }
 
+    pub fn client(&mut self, account_id: &AccountId) -> &mut Client {
+        &mut self.clients[self.account_to_client_index[account_id]]
+    }
+
+    pub fn process_partial_encoded_chunks(&mut self) {
+        let network_adapters = self.network_adapters.clone();
+        for network_adapter in network_adapters {
+            // process partial encoded chunks
+            loop {
+                if let Some(request) = network_adapter.pop() {
+                    match request {
```

Comment:
Heh, I'd write the `if let` as `match`, as it has the `else` branch, and `if let esle` is almost always noisier than the equivalent match. 

On the other hand, this `match` I'd write as `if let`, as it doesn't have an else branch.